examples/three_layer_constant_fluxes.jl

```diff
@@ -392,62 +392,78 @@ tke_budget_statistics = turbulent_kinetic_energy_budget(model, b=b, p=p, U=U, V=
 fields_to_output = merge(model.velocities, model.tracers, (e=e, ϵ=dissipation))
 
 statistics_to_output = merge(primitive_statistics, subfilter_flux_statistics, tke_budget_statistics)
-
-function init_save_some_metadata!(file, model)
-    file["parameters/coriolis_parameter"] = f
-    file["parameters/density"] = 1027.0
-    file["boundary_conditions/θ_top"] = Qᶿ
-    file["boundary_conditions/θ_bottom"] = dθdz_deep
-    file["boundary_conditions/u_top"] = Qᵘ
-    file["boundary_conditions/u_bottom"] = 0.0
-    return nothing
-end
+statistics_to_output = Dict(string(k) => v for (k, v) in statistics_to_output)
 
 @info "Garnishing output writers..."
 
+ρ₀ = 1027
+cₚ = 4000
+
+global_attributes = (
+    name = name,
+    thermocline_type = thermocline_type,
+    reference_density = ρ₀,
+    specific_heat_capacity = cₚ,
```

Comment:
I'm worried about saving some of these parameters in metadata. I think it's confusing to specify parameters that don't affect the dynamics. It misleads people into thinking that the specific heat capacity is a parameter in this simulation. But it's not, because we specify the buoyancy flux, which is the only number that matters. In other words, the reference density, specific heat capacity, gravitational acceleration, and thermal expansion can all be modified independently to produce _identical_ results.

We've suffered from issues like these in the past regarding reproducing other's results. If something besides buoyancy flux is reported, but an incomplete parameter set is given, the buoyancy flux cannot be calculated and the results cannot be reproduced exactly. Usually the missing parameter is something like reference density or specific heat capacity, which only vary by a few % in the ocean and therefore can be "guessed at". But if these parameters can be guessed at, what's the point of recording them in the first place?

I think we should discuss a coherent strategy / framework for these points. Possibly, we should differentiate between "idealized" cases that specify buoyancy flux, and "realistic" cases that use a nonlinear equation of state and require reference densities and gravitational acceleration. Note that if we use TEOS-10 there is only one possible value for specific heat capacity (since we are using conservative temperature --- basically a scaled enthalpy --- as prognostic variable). It might still be a good idea to report specific heat capacity though.